src/Contrib/OtlpGrpc/Exporter.php

```diff
@@ -35,6 +36,17 @@ class Exporter implements SpanExporterInterface
 
     private TraceServiceClient $client;
 
+    private array $retryableStatusCodes = [
+            \Grpc\STATUS_CANCELLED,
+            \Grpc\STATUS_DEADLINE_EXCEEDED,
+            \Grpc\STATUS_PERMISSION_DENIED,
```

Comment:
Not directly related to this PR, but `STATUS_PERMISSION_DENIED` and `STATUS_PERMISSION_DENIED` are incorrect, see [OTLP/gRPC Response](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc-response).

Reply:
@nI can change it in my PR but it was there before I have raised my PR. 
Do you want me to remove in my PR itself?

Reply:
Done